Document/0x05a-Platform-Overview.md

```diff
@@ -579,19 +579,19 @@ Publishing an app is quite straightforward; the main operation is making the sig
 
 ### Android Application Attack surface
 
-The Android application attack surface consists of all components of the application, including the supportive material necessary to release the app and to support its functioning. The Android application can be attacked by:
-- Having unsafe input by means of IPC communication or URL-schemes. See
-  - [Testing for Sensitivie functionality Exposure Through IPC](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-sensitive-functionality-exposure-through-ipc "Testing for Sensitivie functionality Exposure Through IPC");
+The Android application attack surface consists of all components of the application, including the supportive material necessary to release the app and to support its functioning. The Android application may be vulnerable to attack if it does not:
+- Validate all input by means of IPC communication or URL-schemes. See
+  - [Testing for Sensitive functionality Exposure Through IPC](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-sensitive-functionality-exposure-through-ipc "Testing for Sensitive functionality Exposure Through IPC");
   - [Testing URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing URL Schemes").
-- Having unsafe input by the user to input fields.
-- Having unsafe input to a Webview by a user or by having insecure code loaded into the webview. See:
-  -  [Testing javascript execution in webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-javascript-execution-in-webviews "Testing javascript execution in webviews");
+- Validate all input by the user in input fields.
+- Validate the content loaded inside a webview. See:
+  -  [Testing JavaScript execution in webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-javascript-execution-in-webviews "Testing JavaScript execution in webviews");
   - [Testing WebView Protocol Handlers](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-webview-protocol-handlers "Testing WebView Protocol Handlers");
   - [Determining Whether Java Objects Are Exposed Through WebViews]("https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#determining-whether-java-objects-are-exposed-through-webviews "Determining Whether Java Objects Are Exposed Through WebViews").
-- Having insecure responses from a server, or compromised responses by means of a man in the middle attack between the server and the mobile application. See:
+- Securely communicate with backend servers and is susceptible to man-in-the-middle attacks between the server and the mobile application. See:

```

Comment:
should be "and is not susceptible to" then right?

Reply:
My bad!